modules/041-linstor/templates/linstor-affinity-controller/deployment.yaml

```diff
@@ -102,9 +102,9 @@ spec:
           imagePullPolicy: "IfNotPresent"
           args:
             - --leader-election=true
-            - --leader-election-lease-name=$(NAME)
-            - --leader-election-namespace=$(NAMESPACE)
-            - --leader-election-resource-name=linstor-ha-controller
+            - --leader-election-id=$(NAME)
+            - --lease-lock-namespace=$(NAMESPACE)
+            - --lease-lock-name=linstor-affinity-controller
```

Comment:
Why parameters changes ? In new version of linstor-affinity-controller names changes ?

Reply:
I think it always has been. We just didn't noticed that as it is not important component 😞 

https://github.com/piraeusdatastore/linstor-affinity-controller/blob/1414aab05cd32b68b8b04fcde7459b3a944a522b/pkg/leaderelection/leaderelection.go#L32-L34